### PR TITLE
Bring even MOAR attention to the internal Nordstrom Technology Conflu…

### PR DESCRIPTION
**Pull request by gwsii**
…ence configuration page in Lesson 0 and updated page name.

Quick reminder to do a git pull on their repository at the beginning of Lesson 1.
Provided pointer to the Slackbot fan-out hook/unhook configuration and instructions to the fanout.md.

### CODE REVIEW

File: Teachers_Guide/fanout.md [hunk 1]
```diff
@@ -24,3 +24,11 @@ If you use Slack, there is a Slackbot provided in the repo that will allow your
 ./fanout hook --source-arn arn:aws:kinesis:<CoreStreamAWSRegion>:<CoreStreamAWSAccountNumber>:stream/<CoreStreamName> --starting-position TRIM_HORIZON
 ```
 It may take ten minutes before the initial set of records are delivered and the local stream is truly streaming.  The local stream will get only the events that the fanout has processed since the registration was activated for the local stream.  To get the full stream, hook the kinesis stream to the fanout after all local streams have been registered and activated.
+
```
Comment (laurwang) [inline Teachers_Guide/fanout.md]: If you scroll up to just before Step C, there's a sentence about the Slackbot and then a link.  Maybe we can replace that link with a link to this new section instead, something like "See below for details."  It's worth having a section about it to make it more prominent, but I also think it best to make a mention of the Slackbot in the section about registering as well.

File: Lesson0_BeforeWorkshop/README.md [hunk 2]
```diff
@@ -30,12 +30,16 @@ After successfully installing NodeJs, please run the following script provided a
...
 ### Step 3: Setup your AWS credentials
+**_NOTE: Nordstrom Technology!_**
+---
+If you are a _Nordstrom_ engineer, please see the page titled _`Serverless Workshop - Nordstrom Technology Setup`_ in **Confluence** and follow the instructions there.
+
+---
 
-**NOTE:** If you are a _Nordstrom Technology_ engineer, please see the page titled _"Serverless Workshop - Nordstrom AWS Credentials Setup"_ in **Confluence** and follow the instructions there.
 
 Otherwise, install the [AWS-CLI](SETUP-AWS-CLI.md) and use the `aws configure` command to setup your credentials.
```
Comment (laurwang) [inline Lesson0_BeforeWorkshop/README.md]: Is there a way to put the NOTE section a box that is set in a bit from the normal margins of the readme?  There's not a lot of options, but maybe use a quote box (>)?  It's good to call out to the Nordstrom Tech people, but we also want to signal to external users that this box is totally ignorable and separate from the actual project.  

I would at least alter the next sentence to just "Install the AWS-CLI . . . " (drop the "Otherwise").  

I hope that makes sense, bit hard to describe without pointing.

File: Lesson1_HelloRetailStream/README.md [hunk 3]
```diff
@@ -2,6 +2,17 @@
 
 Goal: In order to prevent resource conflicts between participants, you will have your own copy of the hello-retail kinesis event stream in your account.  Once you've created it, we will begin publishing events to it on the day of the conference using a fan-out lambda function on our core stream.
 
+### Step 0: Refresh your repo
+
+This optional step is recommended if you completed [Lesson 0](../Lesson0_BeforeWorkshop/README.md) before the day of the workshop,
```
Comment (laurwang) [inline Lesson1_HelloRetailStream/README.md]: External users won't really have the context of the workshop or the fact we ask people to do Lesson 0 before.  Would rephrase as "This optional step is recommended to make sure you have the latest code, since we make changes often."

**Comment by gwsii** [comment]
Perhaps the best approach would be for you to make the changes you suggest and push to the same branch, rather than having myself make the changes on your behalf (and probably screw it up.) Thanks! :D

**Comment by gwsii** [comment]
Block-quote in Lesson 0 looks great. Good idea!
